modules/node_modules/@ncigdc/containers/ProjectsTable.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
+
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay/compat';
+import { connect } from 'react-redux';
+import { compose } from 'recompose';
+import Showing from '@ncigdc/components/Pagination/Showing';
+import { Row } from '@ncigdc/uikit/Flex';
+import TableActions from '@ncigdc/components/TableActions';
+import entities from '@ncigdc/entities';
+
+import Table, { Tr, Th, Td } from '@ncigdc/uikit/Table';
+
+export const SearchTable = compose(
+  connect(state => ({ tableColumns: state.tableColumns.projects }))
+)(({ downloadable, hits, params, entityType = 'projects', tableHeader, tableColumns, totalsRow }) => {
+  const tableInfo = entities[entityType].table
+    .slice()
+    .sort((a, b) => tableColumns.indexOf(a.id) - tableColumns.indexOf(b.id))
+    .filter(x => tableColumns.includes(x.id));
+
+  return (
+    <div>
+      <Row style={{ backgroundColor: 'white', padding: '1rem', justifyContent: 'space-between', alignItems: 'center' }}>
+        { tableHeader &&
+          <h3 className='panel-title'>
+            {tableHeader}
+          </h3>
+        }
+        <Showing
+          docType={entityType}
```

Comment:
docType should probably stay an API only thing? we don't want to show stuff like `gene-centric` or whatever on the UI